web/core/models.py

```diff
@@ -83,3 +90,28 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+def get_file_name_storage(instance, filename):
+    return os.path.join(str(instance.kid.id), filename)
```

Comment:
A sintaze do `pathlib.Path` ficaria mais legível (e dado que esse projeto não precisa ter suporte a Python 2, melhor, não?):

```suggestion
    return Path(instance.kid.id) / filename
```